ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -248,7 +248,12 @@ then
 fi
 
 load_mon start
-battery_simulator start
+
+if param compare SIM_BAT_ENABLE 1
+then
+	battery_simulator start
```

Comment:
It might make sense to shift this over to px4-rc.simulator. https://github.com/PX4/PX4-Autopilot/blob/22a33ee2361869d520be5f4ec446d58548cab5ba/ROMFS/px4fmu_common/init.d-posix/px4-rc.simulator